smarts/core/scenario.py

```diff
@@ -181,6 +181,7 @@ def __init__(
         social_agents: Dict[str, SocialAgent] = None,
         log_dir: str = None,
         surface_patches: list = None,
+        traffic_history: dict = {},
```

Comment:
In Python, probably not good practice to use `{}` or `[]` as default value. Using `None` is better in this case.